custom_components/ha_creality_ws/camera.py

```
@@ -850,26 +1009,304 @@ async def async_handle_async_webrtc_offer(
+            "ha_creality_ws: direct WebRTC printer offer candidates total=%d video=%d",
+            len(candidates or []),
+            len(video_candidates),
+        )
+        if video_candidates:
+            video = video + video_candidates
+        munged_session = self._replace_bundle(session_lines, ["0"])
+        return self._join_sdp(munged_session, [video])
+
+    def _ha_direct_answer_sdp(self, offer_sdp: str, printer_answer_sdp: str) -> str:
+        """Return an answer SDP whose m-line order matches HA's original offer."""
+        offer_session, offer_sections = self._split_sdp(offer_sdp)
```

Ruff (0.15.15)

[warning] Unpacked variable `offer_session` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)